package.json

```diff
@@ -86,24 +90,27 @@
     "ajv": "^8.11.0",
     "alex": "^11.0.0",
     "eslint": "^8.47.0",
-    "headers-polyfill": "^3.1.2",
+    "headers-polyfill": "^4.0.1",
```

Comment:
Should be able to get rid of this when you move to native `fetch` as well well.

Reply:
One thing to not about using native `fetch` is that you'll need to load in `DOM` and `DOM.Iterable` in the tsconfig. DOM for `fetch` and `DOM.Iterable` for `Headers.entries()`.

Reply:
actually, do you need this library? you're loading `Headers` from `node-fetch` in a couple spots already and that seems to be fine.

Reply:
yeah i'm thinking i'll grab the types piecemeal using this approach: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924#issuecomment-1707342243

Reply:
let me play around, this was technically already an `msw` dep

Reply:
okay yeah swapped it out for `node-fetch` for now! see https://github.com/readmeio/rdme/pull/856/commits/5a2a9e33a7b74392da2a664b12db293a1078f732